e2e/test.js

```diff
@@ -7,6 +7,7 @@ const Browser = require('zombie');
 const browser = new Browser();
 const rollup = require('rollup');
 const tolerance = 1e-4;
+const request = require('request');
```

Comment:
I guess this isn't needed anymore?

Reply:
sorry yes hang over from trying to download straight from the json in js but was just easier (for me) to do this in R
```suggestion

```